### DISCUSSION TITLE
Why option --filename was depricated?

### DISCUSSION THREAD

**Discussion by mexoboy** [go-acme/lego#1782] [Q&A] [answered]
Hi. Have couple question about --filename option. Could you please explain motivation for deprication? We using lego in ansible and in some cases we must known certificate name which will be generated. Option --filename it's perfect for us, but it's deprecated and we not really want to risks using this option. Also could you explain algorithm which using for generation certificate name?

**Comment by ldez** [comment, marked as answer]
https://github.com/go-acme/lego/issues/838#issuecomment-475965177

**Reply by mexoboy** [reply]
Sorry for duplicate. Thanks a lot for answer!
